composer.json

```diff
@@ -16,33 +16,36 @@
         }
     ],
     "require": {
-        "php": "~5.5|~7.0",
-        "sonata-project/admin-bundle": "~3.0",
-        "sonata-project/block-bundle": "^2.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/property-access": "~2.2|~3.0",
-        "doctrine/phpcr-odm": "~1.1",
-        "doctrine/phpcr-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": "~2.0",
-        "symfony-cmf/resource-rest-bundle": "~1.0"
+        "php": "^5.5 || ^7.0",
+        "sonata-project/admin-bundle": "^3.0@dev",
+        "sonata-project/block-bundle": "^3.0@dev",
+        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/property-access": "^2.2 || ^3.0",
+        "doctrine/phpcr-odm": "^1.1",
+        "doctrine/phpcr-bundle": "^1.1",
+        "symfony-cmf/tree-browser-bundle": "^2.0@dev",
+        "symfony-cmf/resource-rest-bundle": "^1.0@dev"
     },
     "require-dev": {
-        "jackalope/jackalope-jackrabbit": "~1.0",
-        "symfony/phpunit-bridge": "~2.7|~3.0",
-        "symfony/security-acl": "~2.4|~3.0",
-        "fabpot/php-cs-fixer": "~0.1|~1.0"
+        "symfony-cmf/resource-bundle": "@dev",
+        "symfony-cmf/resource": "@dev",
+        "dantleech/glob-finder": "@dev",
+        "puli/repository": "@beta",
+        "sonata-project/core-bundle": "@dev",
+        "jackalope/jackalope-jackrabbit": "^1.0",
+        "symfony/phpunit-bridge": "^2.7 || ^3.0",
+        "symfony/security-acl": "^2.4 || ^3.0",
+        "fabpot/php-cs-fixer": "^1.0"
```

Comment:
> Remove `~0.1`, as it makes no sense to install an older version for such a tool (it'll only make Composer slower in resolving versions).

This will not works on lowest dependencies, that's why.

> *Btw, I don't see any reason to add this as a project dependency as nothing relies on this*

This is the old way on it gonna to be changes like we did in https://github.com/sonata-project/SonataAdminBundle/pull/3273.